README.md

```diff
@@ -23,52 +23,90 @@ For complete API documentation, see the [ably documentation](https://ably.io/doc
 
 ## Usage
 
+### With Node.js
+
+For the real-time library:
+
+```javascript
+var realtime = require('ably-js').Realtime;
+```
+
+For the rest-only library:
+
+```javascript
+var rest = require('ably-js').Rest;
+```
+
+### With the Browser library
+
+Include the Ably library in your HTML:
+
+```html
+<script src="https://cdn.ably.io/lib/ably.min.js"></script>
```

Comment:
This should probably include a way to choose your version at some point. At least there should be some guaranty that it won't have breaking changes.